resources/views/entities/components/menu.blade.php

```diff
@@ -2,6 +2,15 @@
 @php $modelMenuItems = $model->menuItems(); @endphp
 <div class="hidden-xs">
 @foreach ($modelMenuItems as $section => $menuItems)
+    @if($section == 2)
```

Comment:
Please mode the logic to the Model and not in the view, so that we can test the code in the future :)